x-pack/plugins/security_solution/scripts/endpoint/common/roles_users/serverless/es_serverless_resources/roles.yml

```diff
@@ -26,84 +25,129 @@ viewer:
   cluster: []
   indices:
     - names:
-        - "/~(([.]|ilm-history-).*)/"
+        - '.siem-signals*'
+        - '.lists-*'
+        - '.items-*'
       privileges:
-        - "read"
-        - "view_index_metadata"
+        - 'read'
+        - 'view_index_metadata'
       allow_restricted_indices: false
     - names:
-        - ".siem-signals*"
-        - ".lists-*"
-        - ".items-*"
+        - '.alerts*'
+        - '.preview.alerts*'
       privileges:
-        - "read"
-        - "view_index_metadata"
+        - 'read'
+        - 'view_index_metadata'
       allow_restricted_indices: false
     - names:
-        - ".alerts*"
-        - ".preview.alerts*"
+        - apm-*-transaction*
+        - traces-apm*
+        - auditbeat-*
+        - endgame-*
+        - filebeat-*
+        - logs-*
+        - packetbeat-*
+        - winlogbeat-*
+        - metrics-endpoint.metadata_current_*
+        - '.fleet-agents*'
+        - '.fleet-actions*'
+        - 'risk-score.risk-score-*'
       privileges:
-        - "read"
-        - "view_index_metadata"
-      allow_restricted_indices: false
+        - read
   applications:
-    - application: "kibana-.kibana"
+    - application: 'kibana-.kibana'
       privileges:
-        - "read"
-      resources:
-        - "*"
+        - feature_ml.read
+        - feature_siem.read
+        - feature_siem.read_alerts
+        - feature_siem.endpoint_list_read
```

Comment:
Can you please add a test for this Role/User (may need to add a user to the `users` and `users_role` files). We keep our serverelss specific tests for Authz here: `x-pack/plugins/security_solution/public/management/cypress/e2e/serverless/roles` . There are tests for both PLI's (essentials and complete). The tests are setup to not only check proper access to allowed features, but also ensures they have no access to others as well.

Reply:
NOTE: we did not have tests for Editor or Viewer before this PR because those roles did not include any Endpoint permissions

Reply:
Never mind on this one. I realized that it was not your PR that actually introduced Viewer/Editor, but an earlier one, so I don't want you to incur the additional changes in this PR